### DISCUSSION TITLE
An ability to create palette from image

### DISCUSSION THREAD

**Discussion by donte5405** [Orama-Interactive/Pixelorama#1141] [Ideas]
How it works
1. User clicks on `Create a palette from image` from palette menu.
2. User picks an image from file browser.
3. The app will pick up all palettes found in the image.
4. The app creates a new palette and put all colours from (2).
5. Done.

Benefits: it's a feature that helps generating palette that helps keeping art consistency.

**Comment by OverloadedOrama** [comment]
Hello, this is already possible by opening an image (either by dragging and dropping it from the file manager into Pixelorama, or by going to File > Open), and selecting "New palette" in the "Import as" option.

https://www.oramainteractive.com/Pixelorama-Docs/user_manual/Import

**Reply by donte5405** [reply]
I figured out right away after posting this, lol. Thanks, regardless!
